packages/opentelemetry-core/package.json

```diff
@@ -6,8 +6,8 @@
   "types": "build/src/index.d.ts",
   "repository": "open-telemetry/opentelemetry-js",
   "scripts": {
-    "test": "nyc ts-mocha -p ./tsconfig.json test/**/*.ts",
-    "tdd": "yarn run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json test/**/*.ts",
```

Comment:
We should attempt to use the built-in v8 coverage reporting in a later revision since it's a lot faster than NYC.